taskcluster/ci/raptor/kind.yml

```diff
@@ -18,6 +18,8 @@ only-for-abis:
     - arm64-v8a
 
 job-defaults:
+    attributes:
+        retrigger: true
     dependencies:
         geckoview-nightly: geckoview-nightly
         linux64-minidump-stackwalk: toolchain-linux64-minidump-stackwalk
```

Comment:
The dependency failure the prints seem to be trying to fix can be fixed by removing this line. I think this would be fixed if you rebase.